### ISSUE TITLE
Switch java_package, java_multiple_files, java_outer_classname to match Google Cloud API format

### ISSUE THREAD

**Issue by peter-edge** [uber/prototool#117]
https://cloud.google.com/apis/design/file_structure

Specifically:

- `java_package` will be `com.PACKAGE` instead of `com.PACKAGE.pb`
- `java_multiple_files` will be set to true
- `java_outer_classname` will be `FILENAMEProto`

`go_package` will not change as there's no guidance on that, and we won't do anything with `csharp_namespace`, `objc_class_prefix`, or `php_namespace` as of yet, as we don't use those and we don't need to overcomplicate this for now (we might want to do `objc_class_prefix` in the future though).

Changes that need to be made:

- Update the linters to enforce this
- Update the create package to create Protobuf files with this
- Update the formatter to set these values after #113 is merged
